signer/src/transaction_coordinator.rs

```diff
@@ -827,6 +859,66 @@ where
         }
     }
 
+    fn authenticate_message(
+        packet: &wsts::net::Packet,
+        public_keys: &hashbrown::HashMap<u32, p256k1::point::Point>,
+        public_key_point: p256k1::point::Point,
+        sender_is_coordinator: bool,
+    ) -> bool {
+        let check_signer_public_key = |signer_id| match &public_keys.get(&signer_id) {
+            Some(signer_public_key) if &public_key_point != *signer_public_key => {
+                tracing::warn!(
+                    ?packet.msg,
+                    reason = "message was signed by the wrong signer",
+                    "ignoring packet"
+                );
+                false
+            }
+            None => {
+                tracing::warn!(
+                            ?packet.msg,
+                            reason = "no public key for signer",
+                signer_id = %signer_id,
+                            "ignoring packet"
+                        );
```

Comment:
nit: rustfmt does not catch the weird spacing here. Also, this could be just
```rust
tracing::warn!(
    ?packet.msg,
    reason = "no public key for signer",
    %signer_id,
    "ignoring packet"
);
```

Reply:
Done